docs/config.json

```
@@ -456,13 +456,32 @@
       	  "slug": "/management/introduction/"
 	},
         {
-          "title": "Admin Guides",
-          "slug": "/management/admin/",
+          "title": "Joining Teleport Services to a Cluster",
```

As is this change causes a line wrap in the nav bar. Suggest something like:

```suggestion
          "title": "Join Teleport Services to a Cluster",
```

---

All of the other titles in this sidebar section are noun phrases, so I've opted for "Joining Teleport Services" for consistency with these